src/webextension/icons/clear.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 48.2 (47327) - http://www.bohemiancoding.com/sketch -->
+    <title>Icon / Custom / Clear</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Form-/-Search-/-DH-Active---Filled" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" transform="translate(-234.000000, -7.000000)">
+        <g id="Form-/-Search---Active-Empty" fill="#b1b1b3">
+            <g id="Icon-/-Custom-/-Clear" transform="translate(234.000000, 7.000000)">
+                <path d="M2.34314575,13.6568542 C-0.781048583,10.5326599 -0.781048583,5.46734008 2.34314575,2.34314575 C5.46734008,-0.781048583 10.5326599,-0.781048583 13.6568542,2.34314575 C16.7810486,5.46734008 16.7810486,10.5326599 13.6568542,13.6568542 C10.5326599,16.7810486 5.46734008,16.7810486 2.34314575,13.6568542 Z M11.7712362,10.6937401 L9.07749605,8 L11.7712362,5.30625988 C12.0687785,5.00871756 12.0687785,4.52630615 11.7712362,4.22876383 C11.4736938,3.93122152 10.9912824,3.93122152 10.6937401,4.22876383 L8,6.92250395 L5.30625988,4.22876383 C5.00871756,3.93122152 4.52630615,3.93122152 4.22876383,4.22876383 C3.93122152,4.52630615 3.93122152,5.00871756 4.22876383,5.30625988 L6.92250395,8 L4.22876383,10.6937401 C3.93122152,10.9912824 3.93122152,11.4736938 4.22876383,11.7712362 C4.52630615,12.0687785 5.00871756,12.0687785 5.30625988,11.7712362 L8,9.07749605 L10.6937401,11.7712362 C10.9912824,12.0687785 11.4736938,12.0687785 11.7712362,11.7712362 C12.0687785,11.4736938 12.0687785,10.9912824 11.7712362,10.6937401 Z" id="Combined-Shape"></path>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
Nit: add a trailing newline

Reply:
In my opinion, I think this class of nit is asking for too much.

While SVGs are technically XML, they are effectively image files created and updated in image processing applications.  Some of those apps add the newline, and some don't, and I don't think it reasonable someone go into the file with a text editor to ensure the newline is present.

Reply:
I was going to say the newline issue would be resolved by running through an SVG linter/optimizer like `svgo`, but now that I look at this more closely, there are bigger issues than just the trailing newline. A quick run through `svgo` shows that we could cut the size of this SVG by 76%.

I'll file an issue to do this for all our existing SVG assets, but it'd be nice to get these new assets optimized. I recommend the following `svgo` config:

```
svgo FILENAME --pretty --indent=2 --disable=removeXMLProcInst
```

I suppose I'd be ok with dropping `--pretty` and `--indent`, but I have occasionally had to look at SVG source to figure out why weird rendering stuff is happening, so having it be readable would be nice.

Reply:
>I'll file an issue to do this for all our existing SVG assets

Want to pile on to this existing one? https://github.com/mozilla-lockbox/lockbox-extension/issues/301